pkg/network/node/node.go

```diff
@@ -425,8 +425,17 @@ func (node *OsdnNode) Start() error {
 		return err
 	}
 
-	if networkChanged && len(existingPods) > 0 {
-		err := node.reattachPods(existingPods)
+	existingPodSandboxes, err := node.getPodSandboxes()
+	if err != nil {
+		return err
+	}
+
+	if err = node.podManager.InitRunningPods(existingPodSandboxes, existingOFPodNetworks); err != nil {
```

Comment:
might be better to put this a little bit above, before the call to `podManager.Start()`?